build.gradle.kts

```diff
@@ -1,11 +1,18 @@
+import com.vanniktech.maven.publish.JavaLibrary
+import com.vanniktech.maven.publish.JavadocJar
+
 plugins {
     id("com.github.spotbugs") version "5.2.5" apply false
-    id("de.aaschmid.cpd") version "3.2"
+// FIXME adapt-publishing cpd fails to run in Gradle 8
+//    id("de.aaschmid.cpd") version "3.2"
 
     id("biz.aQute.bnd") version "6.4.0" apply false
 
-    jacoco
+// FIXME adapt-publishing need replacement for deprecated JacocoMerge
```

Comment:
I think we can get rid of the coverage stuff as well as there is no real active development on this plugin and therefore this quality gate is not really necessary.

Reply:
Seems like https://docs.gradle.org/current/userguide/jacoco_report_aggregation_plugin.html#header is the replacement for it